src/ng-datepicker/ng-datepicker.component.ts

```diff
@@ -227,6 +239,11 @@ export class NgDatepickerComponent implements ControlValueAccessor, OnInit, OnCh
     }
 
     const input = this.elementRef.nativeElement.querySelector('.ngx-datepicker-input');
+
+    if (input == null) {
```

Comment:
`if (!input) {` is better written I think.

Reply:
I don’t think so because if input will be ‘true’ - it will pass this condition

Reply:
okay, I missed that. sorry